OpenRA.Mods.Common/Traits/Buildings/BuildingUtils.cs

```diff
@@ -44,17 +52,18 @@ public static bool CanPlaceBuilding(this World world, string name, BuildingInfo
 				return true;
 
 			var res = world.WorldActor.TraitOrDefault<ResourceLayer>();
+			var actorInfo = world.Map.Rules.Actors[name];
```

Comment:
How much work would it be to pass the `ActorInfo` into this method instead of `name`?

Reply:
I ll look into this.
Still remember you asking for it but i just wanted to make it run first, before improveing.